src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -8,10 +8,16 @@ This is only worth it when the execution is long running, i.e. more than ~50 mil
 """
 directive @async on FIELD_DEFINITION
 
+directive @oneOf on FIELD_DEFINITION
+
 schema {
     query: QueryType
 }
 
+input AccessibilityPreferences {
+    enabled: Boolean
```

Comment:
If this is the root for all accessibility preferences, then I think there should be a WheelchairPreferences inside it with `transit : { unknown : Cost, inaccessible : Cost }` - not a boolean flagg.

Reply:
Enabling "wheelchair" routing with default parameters isn't simple then if there isn't an on/off switch. We don't yet allow changing the accessibility in other ways through the GraphQL API at least. However, adding those options later is not really a problem as long as we have a wrapper type. I agree that it would make sense to already have a wheelchair field with WheelchairPreferences so we can expand the accessibility later on to cover other things as well.

Reply:
I have added a wrapper type for wheelchair in https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/1a5ae1b705db625888856843ec3725a67f095090